index.d.ts

```diff
@@ -0,0 +1,233 @@
+// Entities
+
+export enum ProductType {
+  TRIAL = "TRIAL",
+  DIRECT_SUBSCRIPTION = "DIRECT_SUBSCRIPTION",
+  ONE_TIME = "ONE_TIME"
+}
+
+export enum ProductDuration {
+  WEEKLY = "WEEKLY",
+  MONTHLY = "MONTHLY",
+  THREE_MONTHS = "3_MONTHS",
+  SIX_MONTHS = "6_MONTHS",
+  ANNUAL = "ANNUAL",
+  LIFETIME = "LIFETIME"
+}
```

Comment:
Thank you for your contribution!
Looks like here is a mistake in types
I suggest it would look like this: 

```
export enum ProductDuration {
  WEEKLY = 0,
  MONTHLY = 1,
  "3_MONTHS" = 2,
  "6_MONTHS" = 3,
  ANNUAL = 4,
  LIFETIME = 5,
}
```

Reply:
Updated